### DISCUSSION TITLE
[Suggestion] Ringworld Variants

### DISCUSSION THREAD

**Discussion by Sebateses** [tsunamayo/Starship-EVO#4036] [Suggestions]
Hey Tsuna any plans down the line to make variants of the Ringworld. I did some fanart of the plant guy in a gnarled wasteland. Would be great if someday this game had some real exotic locations. I realize mountains like this in the background may be difficult since I'm assuming the in game ringworld ones the are made by height maps and not assets.

I have a few more Ideas and color schemes in mind that I think could really work well with Starship Evo's all ready vibrant feel.

Let me know what you think. 

![Plant_dude_2](https://user-images.githubusercontent.com/72846347/105244853-32914c00-5bc5-11eb-94be-0226f04b72e0.png)

**Comment by tsunamayo** [comment]
Amazing drawing! So the issue is really a limited time on my side. I need to sort out the terrain tech, then I would be able to think about biome and such. But I want the gameplay loop first. No use to planet / better ringworld if there is nothing to do... Thanks

**Comment by Sebateses** [comment]
I hear ya 100% focus on the gameplay loop. No point if nothing to do in the landscape.

**Comment by tsunamayo** [comment]
Yeah that being said I am doing a lot of tech research on the terrain tech lately. I will need this first for mining - but I want something that will scale for planet tech. By the way your ship are some of the most amazing on the workshop! Kudos to you, those are really some incredible stuff. Still cant wait to give you mech, your still is just soo perfect for mechs... Also mini-drones, that would be perfect. Cheers

**Comment by Kaiser-Indrasil** [comment]
Would love to see Sebateses' stuff highlighted in the Steam update notes ;)

**Comment by Sebateses** [comment]
@tsunamayo thanks alot glad you like it I cannont wait for mechs.
I have a slight issue with decals on my builds which i will get around to making a separate post, I just have to record a video of it. It might just be my graphics settings since I saw it didnt seem to be happening on other pc's.

Again thankyou and so looking forward to mechs!

**Comment by Uncle-Ulty** [comment]
Well, I like the Idea. but some variations could be easier to implement. Maybe a ring of 100% water, 100% plains, 100% hills, or 50% water and 50 hills. You know, just using what is already available in-game... I'm building racing tracks, and some different rings would be awsome for scenario!

**Comment by Sebateses** [comment]
@Uncle-Ulty i like that idea. Maybe the hill one has purple atmosphere with yellow grass and darker mountains. Water one could be like pic with yellow atmosphere and dark blue terrian.

**Comment by Uncle-Ulty** [comment]
Yeah, of course! Changing terrain (with the already implemented textures and shapes) and colors (fog + terrain), would create several variations with what we already have!

I'd love to build big blue over a full water ring!

![image](https://user-images.githubusercontent.com/73253472/105419847-1ec60400-5c1e-11eb-9a9f-c6ca3566e5bd.png)
